tests/foreman/api/test_organization.py

```diff
@@ -246,6 +246,26 @@ def test_positive_update(self, attrs):
             self.assertIn(name, new_attrs.keys())
             self.assertEqual(new_attrs[name], value)
 
+    def test_associate_user_with_organization(self):
+        """@Test: Update an organization, associate user with it.
+
+        @Assert: User is associated with organization.
+
+        @Feature: Organization
+
+        """
+        user_id = entities.User().create()['id']
+
+        client.put(
+            self.organization.path(),
+            {'organization': {'user_ids': [user_id]}},
+            verify=False,
+            auth=get_server_credentials(),
+        ).raise_for_status()
+
+        new_attrs = self.organization.read_json()
+        self.assertEqual(user_id, new_attrs['users'][0]['id'])
```

Comment:
It's also good to check that only one user is associated with `self.organization`. You can make an assertion like this:

```python
attrs = self.organization.read_json()
self.assertEqual(1, len(attrs['users']))
self.assertEqual(user_id, attrs['users'][0]['id'])
```